deploy-metadata.sh

```diff
@@ -1,39 +1,32 @@
 #!/bin/bash
-MMD_IN='s-enda-mmd-xml'
-MMD_XML_REPO="https://gitlab.met.no/mmd/s-enda-mmd-xml"
```

Comment:
For the future, a default value of the variable should point to an example repository on Github.